icaruscode/Filters/CMakeLists.txt

```
@@ -2,6 +2,8 @@ art_make_library()
 set(  MODULE_LIBRARIES
     sbnobj::Common_Trigger
     icarusalg::Utilities
+    sbndaq_artdaq_core::sbndaq-artdaq-core_Overlays_ICARUS
```

I don't think this one is needed, but:
```suggestion
    artdaq_core::artdaq-core_Data
```
may.

---

Yes, likely not